easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -284,6 +284,20 @@ def configure_step(self, srcdir=None, builddir=None):
         # disable CMake user package repository
         options['CMAKE_FIND_USE_PACKAGE_REGISTRY'] = 'OFF'
 
+        # ensure CMake uses EB python, not system or virtualenv python
+        python_root = get_software_root('Python')
+        if python_root:
+            python_version = LooseVersion(get_software_version('Python'))
+            python_exe = os.path.join(python_root, 'bin', 'python')
+            options['Python_EXECUTABLE'] = python_exe
+            # This is needed if someone is still using `find_package(PythonInterp ...)` in their CMakeLists.txt
+            options['PYTHON_EXECUTABLE'] = python_exe
+            # Ensure that both `find_package(Python) and find_package(Python2/3)` work as intended
+            if python_version >= "3":
```

Comment:
Just some IMO better wording as the first option should be below the comment referring to it:
```suggestion
            # This is required for (deprecated) `find_package(PythonInterp ...)`
            options['PYTHON_EXECUTABLE'] = python_exe
            # Ensure that both `find_package(Python) and find_package(Python2/3)` work as intended
            options['Python_EXECUTABLE'] = python_exe
            if python_version >= "3":
```